src/components/Footer/Footer.js

```diff
@@ -63,29 +87,10 @@ const Footer = () => {
           onChange={(evt, proj) => {
             dispatch(setProjection(proj));
           }}
-          projections={[
-            {
-              label: 'CH1093 / LV03',
-              value: 'EPSG:21781',
-              format: c => `${t('Koordinaten')}: ${numberFormat(c)}`,
-            },
-            {
-              label: 'CH1093+ / LV95',
-              value: 'EPSG:2056',
-              format: c => `${t('Koordinaten')}: ${numberFormat(c)}`,
-            },
-            {
-              label: 'Web Mercator',
-              value: 'EPSG:3857',
-              format: c => `${t('Koordinaten')}: ${numberFormat(c)}`,
-            },
-            {
-              label: 'WGS 84',
-              value: 'EPSG:4326',
-              format: c =>
-                `${t('Koordinaten')}: ${c[0].toFixed(5)},${c[1].toFixed(5)}`,
-            },
-          ]}
+          projections={projections}
```

Comment:
How about something like (not tested)?
```suggestion
          projections={[
            projections.find(p => p.value === projection.value,
            ...projections.filter(p => p.value === projection.value)
          ]}
```
This would avoid adding a new prop to MousePosition.

Reply:
You meant:
```suggestion
          projections={[
            projections.find(p => p.value === projection.value),
            ...projections.filter(p => p.value !== projection.value),
          ]}
```
But this would change the order of the values in the drop down list. I didn't want to do this.

Reply:
I agree with @gevos